src/main/webapp/go.css

```diff
@@ -131,3 +131,18 @@ button:hover a {
   font-size: 22px;
   font-weight: bold;
 }
+
+#submit-loader {
```

Comment:
Does the loader need to be hidden by default on the initial page load?

Reply:
Yes -- the window.onload() function in go.js hides the loader when the page first loads.